spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -110,9 +137,17 @@ timeToLive::
 See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html[DynamoDB TTL]
 +
 No default - means no records expiration.
+partitionKey::
+	The partition key name of the table.  This is not configurable.
++
+Value: `KEY`
```

Comment:
That's not correct place to mentioned such a non-configurable property for the target table.
Would be better to mention it in the paragraph with the link before.
Just another sentence after that `Support for consumer groups...` one!